src/abe_policy/tests.rs

```diff
@@ -130,11 +132,15 @@ fn test_edit_policy_attributes() -> Result<(), Error> {
     // Missing dimension remove
     assert!(policy.remove_attribute(&missing_dimension).is_err());
 
-    // Remove all attributes from an dimension
+    // Remove all attributes from a dimension
     policy.remove_attribute(&new_attr)?;
     policy.remove_attribute(&Attribute::new("Department", "HR"))?;
     policy.remove_attribute(&Attribute::new("Department", "MKG"))?;
-    policy.remove_attribute(&Attribute::new("Department", "FIN"))?;
+
+    // TODO: temporary fix before we allow removing an entire dimension
```

Comment:
does it require a new issue on the github?

Reply:
Removing a dimension can only be activated after the addition of neutral points in the dimensions, which is the purpose of the PR #122.

Reply:
Raised issue #132

Reply:
thanks!